specs/proposals.md

```diff
@@ -96,12 +96,23 @@ The exact number is yet to be determined, and will depend on the design of the f
 
 The L2 Output Oracle contract implements the following interface:
 
-```js
+```solidity
```

Comment:
I am against inlining solidity in the specs. We should link to the solidity and have pythonic pseudocode when we need code. Its too easy for things to get out of date

Reply:
I agree, but I just wanted to improve the existing specs. That change is another PR touching several docs.